### PR TITLE
Write up

### PR DESCRIPTION
**Pull request by husky-parul**
Update README at https://github.com/FNNDSC/CHRIS_docs and https://github.com/redhat-university-partnerships/radiology

@danmcp Should I elaborate more on how we scaled pman?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,3 +22,48 @@ ChRIS is comprised of several components and services, and it is designed to hav
...
   * Image Plugin(s) - An image processor. Turns raw image data into something meaningful for the radiologists/doctors. Ex: [FreeSurfer](https://surfer.nmr.mgh.harvard.edu/)
+
+## Usecase
+### Multi-Party Computation
+
+As we know that ChRIS (Children’s Research Integration System) is a web-based medical image platform that allows for various forms of medical image processing (for example, MRIs). To run medical image processing, medical researcher/analyst programs the alogorithm that needs to be executed in form of "plugin". The researcher/analyst submits the plugin along with the data on which the alogorithm is applied via ChRIS store. pfcon, running at BCH datacenter, sends the data via pfioh and the alorithm to be applied on the data to pman. pman and pfioh run in a different datacenter managed by OpenShift on top of OpenStack. pman translates the algorithm programmed in the plugin as Kubernetes job. To add scale to medical image processing, pman divides the job amongst several Kubernetes job pods that run in parallely. OpenShift provides container platform for seamless orchestration and resource management on the cluster.
```
Comment (danmcp) [inline README.md]: ChRIS Research Integration System (also change the other references in this file)

File: README.md [hunk 2]
```diff
@@ -22,3 +22,48 @@ ChRIS is comprised of several components and services, and it is designed to hav
   * [pfioh](https://github.com/FNNDSC/pfioh) - Handles input and output of image data into a data center
```
Comment (danmcp) [inline README.md]: Please update the existing architecture image

**Comment by danmcp** [comment]
Additional requests:

- Update the architecture image to a newer one
- Link to the chris store and the new chris frontend and chris ui design repos

File: README.md (hunk 1 above)
Comment (danmcp) [inline README.md]: As we know, ChRIS

File: README.md [hunk 3]
```diff
@@ -22,3 +22,48 @@ ChRIS is comprised of several components and services, and it is designed to hav
...
   * [pman](https://github.com/FNNDSC/pman) - Handles instantiating and checking the status of image processing jobs
   * Image Plugin(s) - An image processor. Turns raw image data into something meaningful for the radiologists/doctors. Ex: [FreeSurfer](https://surfer.nmr.mgh.harvard.edu/)
+
+## Usecase
```
Comment (danmcp) [inline README.md]: Use Case

Comment (danmcp) [reply README.md]: Actually, just remove this since there is only 1

Comment (husky-parul) [reply README.md]: @danmcp This is ready to merge

File: README.md [hunk 4]
```diff
@@ -16,9 +16,59 @@ ChRIS is comprised of several components and services, and it is designed to hav
...
+ * [ChRIS Store](https://github.com/FNNDSC/ChRIS_store_ui)
+ * [ChRIS Frontend](https://github.com/FNNDSC/ChRIS_ui)
+ * [ChRIS UI Design Repos](https://github.com/fnndsc/cube-design)
+
+
+## Secure Multi-Party Computation with ChRIS
+
+
+### What is Secure Multi-Party Computation (MPC) ?
+![Secure MPC is crypto technique that allows parties to jointly compute a function over their inputs while keeping those inputs private](https://en.wikipedia.org/wiki/Secure_multi-party_computation)
```
Comment (danmcp) [inline README.md]: Remove the ! since this isn't an image link

File: README.md [hunk 5]
```diff
@@ -16,9 +16,59 @@ ChRIS is comprised of several components and services, and it is designed to hav
...
+
+### How does secure MPC fit in Health Care?
+Secure MPC has wide application whenever data is scarce and collaborating over data is useful but restricted by privacy laws. Few examples are:
+* Measuring effectiveness of rehabilitation techniques on gunshot victims
+* Studying rare disease
+
+In both the cases hospitals might want to collaborate over data but they can't share patients' data in the clear due to privacy laws and standard hospital practices. Augmenting ChRIS with cryptographically secure MPC allows multiple hospitals to jointly analyze data that they cannot observe individually. As a consequence, each hospital need not entrust other hospitals or the cloud vendor with its patient's data. To achieve this ChRIS interacts with another open source project ![Conclave Cloud Dataverse or C2D](https://github.com/cici-conclave/conclave-web). C2D uses OpenShift to create isolated computing environment for MPC job pods.
```
Comment (danmcp) [inline README.md]: Remove the ! since this isn't an image link

File: README.md [hunk 6]
```diff
@@ -16,9 +16,59 @@ ChRIS is comprised of several components and services, and it is designed to hav
...
+
+
+## Secure Multi-Party Computation with ChRIS
+
+
+### What is Secure Multi-Party Computation (MPC) ?
+![Secure MPC is crypto technique that allows parties to jointly compute a function over their inputs while keeping those inputs private](https://en.wikipedia.org/wiki/Secure_multi-party_computation)
+
+### How does secure MPC fit in Health Care?
+Secure MPC has wide application whenever data is scarce and collaborating over data is useful but restricted by privacy laws. Few examples are:
```
Comment (danmcp) [inline README.md]: A few examples are:

File: README.md [hunk 7]
```diff
@@ -16,9 +16,59 @@ ChRIS is comprised of several components and services, and it is designed to hav
...
+
+### How are isolated computing environments created in cloud?
+We create several types of isolation by
+* Creating machine like virtualisation using namespace system and SELinux policies that comes by default in Red Hat Enterprise Linux distribution.
+* Using OpenShift projects to allow a community of users to organize and manage their content in isolation from other communities via API authentication.
+* Allowing selective isolation using routes and services. Depending on the level of isolation, pods within a project communicates via services. Communication across different projects are done via routes.
+
+
+### Secure MPC and ChRIS usecase
+![This MPC](https://github.com/FNNDSC/pl-mpcs) plugin compares a patient’s brain volume with the population data collected from two different hospitals.
```
Comment (danmcp) [inline README.md]: Remove the ! since this isn't an image link

**Pull request merged**